openedx/core/djangoapps/programs/tasks.py

```diff
@@ -7,7 +7,7 @@
 from celery.exceptions import MaxRetriesExceededError
 from celery.utils.log import get_task_logger
 from django.conf import settings
-from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
```

Comment:
Fixed this while I was in here. It's bad practice to import the User model directly. Django provides a utility function to import the User model (named `get_user_model`).